docs/updating/updating_to_3.3.md

```diff
@@ -248,6 +248,20 @@ Review the changes to make sure your custom configuration was not affected.
 
 Then, perform a database upgrade relevant to the version you are updating to.
 
+!!! caution "Clear Redis cache"
+    If you are using Redis as your persistence cache storage you should always clear it manually after an upgrade.
+    You can do it in two ways, by using `redis-cli` and executing the following command:
+
+    ```bash
+    FLUSHALL
+    ```
+
+    or by executing the following command:
+
+    ```bash
+    bin/console cache:pool:clear cache.redis
```

Comment:
```suggestion
php  bin/console cache:pool:clear cache.redis
```

Not sure what is convention here

Reply:
I see that this isn't standardized across the documentation (ref: https://doc.ibexa.co/en/latest/guide/search/shop_search/search_indexing/#reindexing vs https://doc.ibexa.co/en/latest/updating/5_update_2.5/#required-manual-cache-clearing-if-using-redis). I think we might start using the full command, so @adamwojs remarks stands in my opinion.